### PR TITLE
replace killablethread with kthread

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
looks good

File: Pipfile [hunk 1]
```diff
@@ -22,6 +22,8 @@ flask-talisman = "==1.0"
 gevent = "==21.12.0"
 gevent-websocket = "==0.10.1"
 taipy-config = {git = "https://git@github.com/avaiga/taipy-config.git@develop"}
+kthread = "==0.2.3"
+werkzeug = "==2.0.3"
```
Comment (FredLL-Avaiga) [inline Pipfile:26]: Why do we need to fix werkzeug version ?

Comment (dinhlongviolin1) [reply Pipfile:26]: in 2.1, they remove 'werkzeug.server.shutdown' so the thread couldnt shut down, which cause the sudden break in everything when stopping flask dev server. This does not affect the application when using gevent.

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**